pkg/apiserver/registry/stats/nodelatencystats/rest.go

```diff
@@ -44,9 +46,10 @@ var (
 )
 
 // NewREST returns a REST object that will work against API services.
-func NewREST() *REST {
+func NewREST(clock clock.Clock) *REST {
```

Comment:
Given that the clock is only necessary for testing, let's use the following approach instead:
```go
func NewREST() *REST {
	return newRESTWithClock(cloc.RealClock{})
}

func newRESTWithClock(clock clock.Clock) *REST {
	return &REST{
		indexer: cache.NewIndexer(cache.MetaNamespaceKeyFunc, cache.Indexers{}),
		clock:   clock,
	}
}
```

This way in apiserver.go you can keep calling `NewREST()` (no clock argument required) and in tests you can call `newRESTWithClock`.